src/ast/mod.rs

```diff
@@ -2411,6 +2414,16 @@ impl fmt::Display for Statement {
                 replace_into,
                 priority,
             } => {
+                let table_name = if let Some(Ident {
+                    value,
+                    quote_style: _,
+                }) = table_alias
+                {
+                    format!("{table_name} AS {value}")
```

Comment:
It seems like we should format the alias using whatever quote style was originally used

Maybe something like

```suggestion
                let table_name = if let Some(table_alias) = table_alias
                {
                    format!("{table_name} AS {table_alias}")
```

Reply:
Thanks, I followed your advice. I rebased and force-pushed. I hope that is ok.